website/docs/user-group-role/access-control/permissions.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "About permissions"
+description: "Learn about global and object permissions in authentik."
+---
+
+Permissions are the central components in all access control systems, the lowest-level component, the controlling piece of access data. Permissions are assigned to (or removed from!) to define exactly WHO can do WHAT to WHICH part of the overall software system.
+
+## Fundamentals of authentik permissions
+
+There are two main types of permissions in authentik:
+
+-   [**Global permissions**](#global-permissions)
+-   [**Object permissions**](#object-permissions)
+
+### Global permissions
+
+Global permissions define who can do what on a global level, across the entire system. Some examples in authentik are the ability to add new [flows](../../flow/index.md) or to create a URL for users to recover their login credentials.
+
+You can assign _global permissions_ to individual [users](../user/index.mdx) or to [roles](../roles/index.mdx) (most common and best practice).
+
+### Object permissions
+
+Object permissions have two categories:
+
+-   **_User_ object permissions**: defines WHO (which user) can change the **_object_**
+-   **_Role_ object permissions**: defines which ROLE can change the **_object_**
+
```

Comment:
Suggestion: Remove the parenthetical.  Use parallelism to improve the impact.

```
-  **_User_ object permissions**: defines which USER can change the **_object_**
```

I feel like this is conceptually disconnected, because we eventually learn that roles are assigned to groups, then users are assigned to groups, and those users inherit the permissions of the roles in the group.

Reply:
Agree with Ken, it should be clearer it still defines who can access what, just with the abstraction of roles and groups in the middle